### PR TITLE
FIX Wrong return values in GET requests

### LINKED ISSUE #23: Wrong result for device group creation
When device groups are created through the manager, the response should be an empty object, instead of an array of options as the one currently returned.

### PR DESCRIPTION
**Pull request by dmoranj**
Fixes #23

### CODE REVIEW

File: lib/services/iotaRedirector.js [hunk 1]
```diff
@@ -209,7 +216,7 @@ function processRequests(req, res, next) {
 
             if (statusCodes.length === 1) {
                 if (results.length === 1) {
-                    res.status(statusCodes[0]).json(results[0]);
+                    res.status(statusCodes[0]).json(results[0][1]);
```
Comment (AlvaroVega) [inline lib/services/iotaRedirector.js]: How sure are you about results is a matrix? Already results[0] is checked, but results[0][1] exists?

Comment (dmoranj) [inline lib/services/iotaRedirector.js]: I am. As you can see, the results attribute is a result from a `map()` function (that is a vector) whose elements are the result of the `sendRequest()' function (always vectors, as you can see in line 203).

**Comment by gtorodelvalle** [comment]
LGTM

**Comment by AlvaroVega** [comment]
LGTM

**Pull request merged**